service/mock.py

```diff
@@ -356,3 +359,81 @@ def attach_volume(self, *args, **kwargs):
 
     def detach_volume(self, *args, **kwargs):
         raise NotImplementedError()
+
+
+class MockAccountDriver(BaseAccountDriver):
+
+    def __init__(self):
+        self.glance_images = ALL_GLANCE_IMAGES
+        self.project_name = "admin"
+        self.GlanceImage = warlock.model_factory(GLANCE_IMAGE_SCHEMA)
+        return
+
+    def get_project_by_id(self, project_id):
+        PROJECT_SCHEMA = {'name': {'type': ['null', 'string'], 'description': 'name of the project/tenant', 'maxLength': 255}}
+        MockProject = warlock.model_factory(PROJECT_SCHEMA)
+        if not project_id:
+            return None
+        elif 'admin' in project_id.lower():
+            return MockProject(name='admin')
+        else:
+            return MockProject(name='UnknownUser')
+
+    def _generate_glance_image(self, **kwargs):
```

Comment:
Also unsure how this is related

Reply:
The MockAccountDriver allows me to [write test cases](https://github.com/cyverse/atmosphere/pull/532/files#diff-968ed44e82322a9aca620da73c70a50fR19) that test images using the same logic as `monitor_machines_for` to ensure we get the desired results.